test/e2e/e2e.go

```diff
@@ -153,6 +154,9 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// Reference common test to make the import valid.
 	commontest.CurrentSuite = commontest.E2E
 
+	// Reference federation test to make the import valid.
+	federationtest.FederationSuite = commontest.FederationE2E
```

Comment:
This variable `federationtest.FederationSuite` is created and called just so that the e2e_federation package is included in e2e package during compilation. It wouldn't have any impact at runtime, so using this variable inside if FEDERATION=true will not be of any use.